api/ServiceWorkerMessageEvent.json

```diff
@@ -0,0 +1,367 @@
+{
+  "api": {
+    "ServiceWorkerMessageEvent": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/ServiceWorkerMessageEvent",
+        "support": {
+          "webview_android": {
+            "version_added": false
+          },
+          "chrome": {
+            "version_added": "45"
+          },
+          "chrome_android": {
+            "version_added": "45"
+          },
+          "edge": {
+            "version_added": "17",
+            "notes": "Service workers is available in Microsoft Edge starting EdgeHTML 16 behind a flag."
```

Comment:
"flags"

Reply:
Hmm. As the data in the table & notes only refers to the above, how do I shape this as a flag? Previously (Fx example earlier today) I've had:

```
"type": "preference",  // known
"name": "dom.payments.request.enabled",  // unknown
"value_to_set": "true" // unknown
```
Does it get added as a note? Looking in Edge release notes, the flag is not clear to a newbie like me. :confused:

Reply:
ahh I think you answered this elsewhere. Wrapping my head around it. No need to respond to me now.

Reply:
Yes, this is it (indentation not correct):

"edge": [ {
"version_added": "17"
},
{
"version_added: "16",
"flags": [ { 
   "type": "preference",
    "name": "Enable service workers"
 } ]
}
]